agent/util-scripts/pbench-tool-data-sink

```diff
@@ -299,8 +298,7 @@ class ToolDataSink(Bottle):
         self.web_server_thread = None
 
     def run(self):
-        """run - Start the Bottle web server running and the watcher thread.
-        """
+        """run - Start the Bottle web server running and the watcher thread."""
```

Comment:
The human brain is a weird thing. I skimmed past this without paying attention, and my brain told me it was something about a bottle and watching the water run...

OK, maybe it's just me. ;-)